examples/test_error.py

```diff
@@ -160,7 +155,7 @@
     # }}}
 
     # {{{ Calculating propagated error along active CT on noise slice
-    active_error,_ = active_propagation(s, signal_pathway, indirect="nScans")
+    active_error = active_propagation(s, signal_pathway, indirect="nScans")
```

Comment:
don't know what this was -- it broke the script for me.

Reply:
weird, mine did not run without that... active_propagation on my computer wanted to return 2 things, but we only cared about the first one

Reply:
Does it run for you now?

Reply:
It does. I do not know why` active_propagation` it was returning `frq_slice` but it no longer is